scopes/workspace/workspace/bit-map.ts

```diff
@@ -191,14 +191,23 @@ export class BitMap {
     this.legacyBitMap.components.forEach((componentMap) => {
       const config = componentMap.config;
       if (!config) return;
+
       Object.keys(config).forEach((aspectId) => {
-        const fullSourceId = this.getBitmapEntry(sourceId).id.toStringWithoutVersion();
+        let fullSourceId: string;
+        try {
+          fullSourceId = this.getBitmapEntry(sourceId).id.toStringWithoutVersion();
```

Comment:
Instead of try catch you can use 
`getBitmapEntryIfExist`
And instead of doing `toStringWithoutVersion`
you can pass a second arg to the get like this:
{
  "ignoreVersion": true
}

Reply:
Thanks 
Actually, I realized that the sourceId is already a componentId so this whole get entry is just redundant.